src/redux/slices/educational-instutions/middleware.ts

```diff
@@ -78,10 +83,60 @@ const clearInstitutesList = () => async (dispatch: AppDispatch) => {
   }
 }
 
+const createEducationalInstitutes =
+  (formData: ICreateEducationalInstituteFormDataProps) => async (dispatch: AppDispatch) => {
+    try {
+      dispatch(setCreateIndividualInstitutes(true))
+      await API.educationalInstitutes.createEducationalInstitute(formData)
+    } catch (error) {
+      /* empty */
+    }
+  }
+
+const deleteEducationalInstitute = (uuid: string) => async (dispatch: AppDispatch) => {
+  try {
+    await API.educationalInstitutes.deleteEducationalInstitute(uuid)
+
+    const filters = {
+      page: 1,
+      filters: [],
+    }
+
+    dispatch(educationalInstitutesMiddleware.clearInstitutesList())
+    dispatch(educationalInstitutesMiddleware.getEducationalInstitutes(filters))
+  } catch (error) {
+    /* empty */
+  }
+}
+
+const resetCreateIndividualInstitutesSuccess = () => (dispatch: AppDispatch) => {
+  dispatch(setCreateIndividualInstitutes(false))
+}
+
+const changeEducationalInstitutes =
+  (formData: IChangeEducationalInstituteFormDataProps) => async (dispatch: AppDispatch) => {
+    try {
+      await API.educationalInstitutes.changeEducationalInstitute(formData)
+
+      dispatch(setChangeIndividualInstitutes(true))
+    } catch (error) {
+      /* empty */
```

Comment:
same

Reply:
we have global error, in middleware we leave catch empty